src/store/modules/slide/index.js

```diff
@@ -252,18 +252,6 @@ const mutations = {
   [CURRENT_SLIDE.SET_META_ENDDATE] (state, payload) {
     state.currentSlide.meta.endDate = payload
     state.isCurrentSlideDirty = true
-  },
-
-  // takes a template object as payload. set template of currentslide, and dirty it.
-  [CURRENT_SLIDE.SET_TEMPLATE] (state, payload) {
-    state.currentSlide.template = payload
-    state.isCurrentSlideDirty = true
-  },
-
-  // takes timeout as payload (int), and sets timeout for currentslide, dirty it.
-  [CURRENT_SLIDE.SET_TIMEOUT] (state, payload) {
-    state.currentSlide.timeout = payload
-    state.isCurrentSlideDirty = true
```

Comment:
Are these supposed to be removed?

Reply:
Yes these were missed by a merge, they shouldn't exist in the mutations according to the new slide object schema.

Reply:
Should thus be removed from the pr then?

Reply:
You can keep them there, because master has them, and that is not correct. So you are right to include the removal of these two mutations as part of this PR